bftengine/src/bftengine/ReplicaImp.hpp

```diff
@@ -585,6 +586,11 @@ class ReplicaImp : public InternalReplicaApi, public ReplicaForStateTransfer {
 
   void setConflictDetectionBlockId(const ClientRequestMsg&, IRequestsHandler::ExecutionRequest&);
 
+  inline bool isCreateDbCheckPtSeqNumSet(const SeqNum& seqNum) {
```

Comment:
Can we move this to DbCheckPointManager. This api is not related to ReplicaImp class

Reply:
Moved into DbCheckPointManager